File: B74560/Laboratorios/Lab1/Orbis/src/main/java/com/orbis/B74560/business/EditorialBusiness.java
package com.orbis.B74560.business;

import java.util.Iterator;
import org.springframework.beans.factory.annotation.Autowired;

import com.orbis.B74560.data.EditorialData;
import com.orbis.B74560.domain.Editorial;


public class EditorialBusiness {

	@Autowired
	private EditorialData editorialDao;
	
	public Iterator<Editorial> buscarPorEditorial(int id_editorial) {
		return editorialDao.buscarPorEditorial(id_editorial);
	}
	
	public Iterator<Editorial> buscarEditorialRango(int rangoInicio,int rangoFinal) {
		return editorialDao.buscarEditorialRango( rangoInicio, rangoFinal);
	}
	
}

File: B63685/Labs/LabUnoASPNET/OrbisB63685Model/Business/EditorialBusiness.cs
using OrbisB63685Model.Data;
using OrbisB63685Model.Domain;
using System;
using System.Collections.Generic;
using System.Text;

namespace OrbisB63685Model.Business
{
    public class EditorialBusiness
    {
        EditorialData editorialData = new EditorialData();

        public IEnumerable<Editorial> GetAll()
        {

            return editorialData.GetAll();
        }


    }
}

